Sources/Controllers/TargetMenu/OACollapsableNearestPoiTypeView.h

```diff
@@ -0,0 +1,22 @@
+//
+//  OACollapsableNearestPoiTypeView.h
+//  OsmAnd
+//
+//  Created by nnngrach on 17.08.2021.
+//  Copyright © 2021 OsmAnd. All rights reserved.
+//
+
+#import "OACollapsableView.h"
+
+@class OAPOIType;
+@class OAPOI;
+@class OAPOIUIFilter;
+
+@interface OACollapsableNearestPoiTypeView : OACollapsableView
+
+@property (nonatomic, readonly) NSMutableArray<OAPOIType *> *poiTypes;
```

Comment:
Poi types are not mutated anywhere, so they should be immutable. Also, double-check if you need these properties to be public.